models/nlp/qa/huggingface/inference.py

```diff
@@ -0,0 +1,83 @@
+import csv
+import zipfile
+
+import torch
+import valohai
+from transformers import AutoModelForQuestionAnswering, AutoTokenizer
+
+from utils.serializers import get_serializer
+
+
+def load_model(model_path):
+    tokenizer = AutoTokenizer.from_pretrained(model_path)
+    model = AutoModelForQuestionAnswering.from_pretrained(model_path)
+    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+    model.to(device)
+    return tokenizer, model, device
+
+
+def load_data(data_path):
+    data = []
+    with open(data_path) as csvfile:
+        reader = csv.DictReader(csvfile)
+        for row in reader:
+            data.append(
+                {
+                    "id": row["id"],
+                    "context": row["context"],
+                    "question": row["question"],
+                },
+            )
+    return data
+
+
+def predict(question, tokenizer, model, device):
+    inputs = tokenizer(question["question"], question["context"], return_tensors="pt")
+    inputs = {key: tensor.to(device) for key, tensor in inputs.items()}
+    with torch.no_grad():
```

Comment:
Why aren't the prediction invocations in the other examples using `torch.no_grad()`?